modules/apps/27-interchain-accounts/controller/ibc_module_test.go

```diff
@@ -462,12 +462,70 @@ func (suite *InterchainAccountsTestSuite) TestOnRecvPacket() {
 
 			tc.malleate() // malleate mutates test data
 
-			module, _, err := suite.chainB.App.GetIBCKeeper().PortKeeper.LookupModuleByPort(suite.chainB.GetContext(), path.EndpointB.ChannelConfig.PortID)
+			module, _, err := suite.chainA.App.GetIBCKeeper().PortKeeper.LookupModuleByPort(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID)
```

Comment:
this was routing to the host submodule instead of the controller and thus was missing code coverage

Reply:
good spot!